go/vt/vtgate/vtgate.go

```diff
@@ -238,10 +233,8 @@ func (vtg *VTGate) Execute(ctx context.Context, sql string, bindVariables map[st
 	}
 
 	// Normal processing.
-	startTime := time.Now()
-	ltt := topoproto.TabletTypeLString(tabletType)
-	statsKey := []string{"Execute", "Any", ltt}
-	defer vtg.timings.Record(statsKey, startTime)
+	statsKey := []string{"Execute", session.TargetString, "NA"}
```

Comment:
"Not Applicable". The V2 APIs populate that field, but V3 doesn't need to. Maybe we should consider a different stat variable.